src/kvasir/mpl/algorithm/group.hpp

```diff
@@ -0,0 +1,80 @@
+//          Copyright Jonathan Poelen 2017.
+// Distributed under the Boost Software License, Version 1.0.
+//     (See accompanying file LICENSE.md or copy at
+//          http://www.boost.org/LICENSE_1_0.txt)
+#pragma once
+
+#include "../algorithm/fold_left.hpp"
```

Comment:
many of these includes seem redundant

Reply:
I have not made any assumptions about files used by others. except bind.hpp and call.hpp, the others ask for an investigation, is there any rule?